travis-ci/run-standard-tests.sh

```diff
@@ -25,6 +25,25 @@ echo
 display_and_run $GOVETCMD ./...
 echo
 
+# Environment
+echo "*** Setting up test environment"
+if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+    if [[ "$TRAVIS_SUDO" == true ]]; then
+        # Ensure that IPv6 is enabled.
+        # While this is unsupported by TravisCI, it still works for localhost.
+        sudo sysctl -w net.ipv6.conf.lo.disable_ipv6=0
+        sudo sysctl -w net.ipv6.conf.default.disable_ipv6=0
+        sudo sysctl -w net.ipv6.conf.all.disable_ipv6=0
+    fi
+else
+    if [[ "$TRAVIS_SUDO" == true ]]; then
+        # OSX has a default file limit of 256, for some tests we need a
+        # maximum of 8192.
+        sudo launchctl limit maxfiles 8192 8192
+        ulimit -n 8192
```

Comment:
I am not sure why `sudo launchctl limit maxfiles 8192 8192` is needed but I would think `ulimit -n 8192` should always be used on mac osx, unless it does not do anything without `sudo launchctl limit maxfiles 8192 8192`.

Reply:
I was assuming that I needed to change the global maximum, but it looks like it is already set at unlimited. The launchctl limit line is apparently unnecessary, and ulimit could be used alone.

Reply:
@Coderlane do you want to create another P.R. to remove the sudo check for OsX and just use `ulimit -n 8192`?